.github/workflows/test.yml

```diff
@@ -28,217 +34,105 @@ jobs:
-        uses: ./DDS-Router/.github/actions/run-test-windows
-        with:
-          package_name: ddsrouter_utils
-          cmake_build_type: "Release"
-        if: always()
-
-      - name: Run tests ddsrouter_event
-        uses: ./DDS-Router/.github/actions/run-test-windows
-        with:
-          package_name: ddsrouter_event
-          cmake_build_type: "Release"
-        if: always()
-
-      - name: Run tests ddsrouter_core
-        uses: ./DDS-Router/.github/actions/run-test-windows
-        with:
-          package_name: ddsrouter_core
-          cmake_build_type: "Release"
-        if: always()
-
-      - name: Run tests ddsrouter_yaml
-        uses: ./DDS-Router/.github/actions/run-test-windows
-        with:
-          package_name: ddsrouter_yaml
-          cmake_build_type: "Release"
-        if: always()
-
-      - name: Run tests ddsrouter_tool
-        uses: ./DDS-Router/.github/actions/run-test-windows
-        with:
-          package_name: ddsrouter_tool
-          cmake_build_type: "Release"
-        if: always()
-
-  windows-build-test-debug:
-    runs-on: windows-latest
-    env:
-      CXXFLAGS: /MP
-      OPENSSL64_ROOT: "C:/Program Files/OpenSSL-Win64"
-
-    steps:
-      - name: Sync eProsima/DDS-Router repository
-        uses: actions/checkout@v2
-        with:
-          path: DDS-Router
-
-      - name: Install OpenSSL
-        uses: ./DDS-Router/.github/actions/install-openssl-windows
-
-      - name: Install GTest
-        uses: ./DDS-Router/.github/actions/install-gtest-windows
-        with:
-          cmake_build_type: "Debug"
+          cmake_build_type: ${{ matrix.cmake-config }}
 
       - name: Install yaml-cpp
         uses: ./DDS-Router/.github/actions/install-yamlcpp-windows
         with:
-          cmake_build_type: "Debug"
-
-      - name: Install foonatham memory
-        uses: ./DDS-Router/.github/actions/install-foonathan-memory-windows
-        with:
-          cmake_build_type: "Debug"
+          cmake_build_type: ${{ matrix.cmake-config }}
 
-      - name: Install Fast CDR
-        uses: ./DDS-Router/.github/actions/install-fastcdr-windows
-        with:
-          cmake_build_type: "Debug"
+      - name: Get build eProsima dependencies Job Id
+        shell: pwsh
+        run: |
+          $RUNS_URI = '${{ github.api_url }}/repos/${{ github.repository }}/actions/workflows/build_fastdds.yml/runs';
+          Invoke-RestMethod -Uri $RUNS_URI -OutFile content.json;
+          $job_id = jq -c '.workflow_runs[]  | select(.status == \"completed\") | .id' .\content.json | select -first 1;
+          echo "fastdds_job_id=$job_id" >> $env:GITHUB_ENV;
 
-      - name: Install Fast DDS
-        uses: ./DDS-Router/.github/actions/install-fastdds-windows
+      - name: Download eProsima dependencies
+        uses: dawidd6/action-download-artifact@v2
         with:
-          cmake_build_type: "Debug"
+          workflow: build_fastdds.yml
+          path: ${{ github.workspace }}\..\fastdds\install
```

Comment:
why is this relative path used?

Reply:
Everything is installed in `$GITHUB_WORKSPACE` and the colcon command is run from there. I moved the fastdds directory with the binaries to the parent directory to not mix-up the installation.